src/locale/locales/pt-BR/messages.po

```diff
@@ -1716,7 +1716,7 @@ msgstr "Criar uma nova conta"
 
 #: src/components/StarterPack/ShareDialog.tsx:158
 #~ msgid "Create QR code"
-#~ msgstr ""
+#~ msgstr "Criar QR code"
```

Comment:
In other places, you suggest using `Código QR` as a translation for `QR code`. I myself use `QR code`, but I might biased since I am also in tech, and mixing English and Portuguese is what we do :) My point is: let's make them all `código QR` or all `QR code` instead of mixing the translation approaches

Reply:
It's confusing, but I did a quick survey with my Brazilian relatives. And we use more QR codes. I'm going to adjust them all to English 'QR code'.